setup.py

```diff
@@ -12,7 +12,9 @@
         "joblib>=0.16",
```

Comment:
Should we also have an `environment.yml` to make it easier to configure the run environment? The new MolSSI Cookiecutter requires this, so we can wait until we migrate.

Reply:
Definitely on board with this! I'm having an issue with conda currently (my version doesn't have the `--from-history` option for `conda env export`, and the newer versions seem broken for my OS). I'll plan to add it to a later PR, or when we do the migration.